src/types/records-method.ts

```diff
@@ -0,0 +1,24 @@
+import type { GenericMessage, GenericSignaturePayload } from '../types/message-types.js';
+
+/**
+ * A signer that is capable of generating a digital signature over any given bytes.
```

Comment:
copy-paste error. Also, what is the reason for `RecordsMethod` interface? It looks like any `Message` will satisfy this interface.

Reply:
Is part of the intention to have `RecordsQuery` and `RecordsRead` implement `RecordsMethod`? Looks like only `RecordsWrite implements RecordsMethod`.

Reply:
> It looks like any Message will satisfy this interface.

It does for the most part with the exception that `RecordsWrite` can't extend `Message` easily due to its actual implementation. So the intention is to have the newly defined "real" interface act as the true type (as programming interface intends) when passing the variable instead of depending on an implementation (`Message`).

`RecordsQuery`, and `RecordsRead` still indirectly implements `RecordsMethod` though `Message`.

I originally was thinking to create the new interface just for `Records` related methods, but looks like it can (and should) be renamed to something more generic across DWN Interface, so I am now calling it `InterfaceMethod` instead... not happy with the name at all, so open to suggestions.

Reply:
Actually... really don't like the names, gonna change it.